File: Estrutura_Repetição/C/Ex1080/Main.c
#include <stdio.h>

int main() {
    int valores[100];
    int maior = 0;
    int posicao = 0;

    for (int i = 0; i < 100; i++) {
        scanf("%d", &valores[i]);

        if (valores[i] > maior) {
            maior = valores[i];
            posicao = i + 1;
        }
    }

    printf("%d\n", maior);
    printf("%d\n", posicao);

    return 0;
}

File: Estrutura_Repetição/Java/Ex1080/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int[] valores = new int[100];
        int maior = 0;
        int posicao = 0;

        for (int i = 0; i < 100; i++) {
            valores[i] = sc.nextInt();

            if (valores[i] > maior) {
                maior = valores[i];
                posicao = i + 1;
            }
        }

        System.out.println(maior);
        System.out.println(posicao);
    }
}